rust/utils/run-locally/src/config.rs

```diff
@@ -6,6 +6,7 @@ pub struct Config {
     pub ci_mode: bool,
     pub ci_mode_timeout: u64,
     pub kathy_messages: u64,
+    // TODO: Include count of sealevel messages in a field separate from `kathy_messages`?
```

Comment:
is this needed?

Reply:
Yep, because the sealevel message count is separate from evm message count and set in the test logic here: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2585#discussion_r1310024392